Parser/readWQMdat.m

```diff
@@ -1,438 +1,449 @@
-  sample_data.dimensions{1}.data            = sample_data.dimensions{1}.typeCastFunc(time);
-  
-  sample_data.variables{end+1}.name           = 'TIMESERIES';
-  sample_data.variables{end}.typeCastFunc     = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-  sample_data.variables{end}.data             = sample_data.variables{end}.typeCastFunc(1);
-  sample_data.variables{end}.dimensions       = [];
-  sample_data.variables{end+1}.name           = 'LATITUDE';
-  sample_data.variables{end}.typeCastFunc     = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-  sample_data.variables{end}.data             = sample_data.variables{end}.typeCastFunc(NaN);
-  sample_data.variables{end}.dimensions       = [];
-  sample_data.variables{end+1}.name           = 'LONGITUDE';
-  sample_data.variables{end}.typeCastFunc     = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-  sample_data.variables{end}.data             = sample_data.variables{end}.typeCastFunc(NaN);
-  sample_data.variables{end}.dimensions       = [];
-  sample_data.variables{end+1}.name           = 'NOMINAL_DEPTH';
-  sample_data.variables{end}.typeCastFunc     = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-  sample_data.variables{end}.data             = sample_data.variables{end}.typeCastFunc(NaN);
-  sample_data.variables{end}.dimensions       = [];
-
-  % create a variables struct in sample_data for each field in the file
-  % start index at 4 to skip serial, date and time
-  isUmolPerL = false;
-  
-  for k = 4:length(fields)
-
-    [name, comment] = getParamDetails(fields{k}, params);  
-    data = samples{k-1};
-            
-    coordinates = 'TIME LATITUDE LONGITUDE NOMINAL_DEPTH';
-    
-    % dimensions definition must stay in this order : T, Z, Y, X, others;
-    % to be CF compliant
-    sample_data.variables{end+1}.dimensions         = 1;
-    sample_data.variables{end}.name                 = name;
-    sample_data.variables{end}.typeCastFunc         = str2func(netcdf3ToMatlabType(imosParameters(sample_data.variables{end}.name, 'type')));
-    sample_data.variables{end}.data                 = sample_data.variables{end}.typeCastFunc(data);
-    sample_data.variables{end}.coordinates          = coordinates;
-    sample_data.variables{end}.comment              = comment;
-    
-    % WQM uses SeaBird pressure sensor
-    if strncmp('PRES_REL', sample_data.variables{end}.name, 8)
-        % let's document the constant pressure atmosphere offset previously 
-        % applied by SeaBird software on the absolute presure measurement
-        sample_data.variables{end}.applied_offset = sample_data.variables{end}.typeCastFunc(-14.7*0.689476);
+function sample_data = readWQMdat(filename, mode)
+    %readWQMdat parses a .dat file retrieved from a Wetlabs WQM instrument.

```

Comment:
Again, new indent everywhere.

Reply:
as the commit says this is a linter pass. The entire commit is just new indentation since the file did not follow any indentation.